### PR TITLE
Successful trip preferences axios post  to DB

### PR DESCRIPTION
**Pull request by gjones2019**

### CODE REVIEW

File: client/components/preferences.js [hunk 1]
```diff
@@ -17,8 +17,7 @@ const useStyles = makeStyles({
 });
 
 // exports our ContinuousSlider
-export default function ContinuousSlider(props) {
-  const { currentUser, currentId } = props;
+export default function ContinuousSlider({ currentUser }) {
```
Comment (jstrzesz) [inline client/components/preferences.js:20]: Change this to a function expression and move the export default to the end of the file

**Review by jstrzesz — DISMISSED** [review]
see comments

File: client/components/Splash.js [hunk 2]
```diff
@@ -1,5 +1,4 @@
 import React from 'react';
-import PropTypes from 'prop-types';
```
Comment (jstrzesz) [inline client/components/Splash.js:2]: why are you removing PropTypes?

File: client/components/preferences.js (hunk 1 above)
Comment (gjones2019) [reply client/components/preferences.js]: Changed it

File: client/components/Splash.js (hunk 2 above)
Comment (gjones2019) [reply client/components/Splash.js:2]: It wasn't being used. Jessa said to, it was giving us and error in the console bcus it wasn't being used

**Review by joshjnunez — APPROVED** [review]

**Review by jldaggs88 — APPROVED** [review]

**Pull request merged**